src/Plugin/Components/SettingsContainer.ts

```diff
@@ -185,6 +169,11 @@ export class SettingsContainer {
 					dropdown.addOption("", "Select Style");
 					dropdown.addOption("natural", "Natural");
 					dropdown.addOption("vivid", "Vivid");
+
+					dropdown.onChange((change: "vivid" | "natural") => {
```

Comment:
Random or intentional double new line here?

Reply:
probably not intentional, will look into it.